### ISSUE TITLE
Sketch 3.4 crashes with some types of text layers when exporting

### ISSUE THREAD

**Issue by rychlis** [avocode/avocode#497]
originally reported by @liodub in #472:
```
sketch 3.4 and avocode sketch plug in 212
Dowloaded the fix, clean install of the plugin, still getting the crash wile trying to sync

problem appears to be with font:
single art board with just a shape on it seems to works. 
single art board with text, sketch crashes.

Please advise. Lost work time narrowing down the issue. Im under lot if pressure to deliverer those design to the dev team. Thanks in advance for your help.```

**Comment by rychlis** [comment]
after some investigation we found out this is caused by internal change in Sketch objc representation of test layers with Bullet list type.

**Comment by liodub** [comment]
right, i do have bullet list formatting on some of the artboards.

what would be the safest route to go now?
upload to latest sketch, latest extension, remove bullet and sync

or better to stay with sketch 3.3.3. for now with the latest version of the
extension?

thanks a lot for your help.

**Comment by rychlis** [comment]
Yes basically as you said, your options are:
1. Remove all layers with text lists and sync with Sketch 3.4 (probably best if you don't have much layers of this type)
2. Use Sketch 3.3.3 with the latest version of extension
3. Wait for us to release a fixed version of the extension which should work fine even with Sketch 3.4, hopefully this will be released soon

**Comment by rychlis** [comment]
We've just released 2.1.4 which should fix the crashing! https://manager.avocode.com/downloads/#plugins

**Comment by liodub** [comment]
brilliant! thanks a lot.
